src/Microsoft.SqlTools.ServiceLayer/DacFx/DacFxService.cs

```diff
@@ -286,6 +287,27 @@ public async Task HandleGenerateTSqlModelRequest(GenerateTSqlModelParams request
             }
         }
 
+        /// <summary>
+        /// Handles request to get objects from sql model
+        /// </summary>
+        /// <returns></returns>
+        public async Task HandleGetObjectsFromTSqlModelRequest(GetObjectsFromTSqlModelParams requestParams, RequestContext<TSqlObjectInfo[]> requestContext)
+        {
+            TSqlObjectInfo[] objectInfos = { };
+            var model = projectModels.Value[requestParams.ProjectUri];
+
+            if (model == null)
+            {
+                await requestContext.SendError(new Exception($"Could not find SQL model from project {requestParams.ProjectUri}"));
```

Comment:
Add a return here to avoid any potential issues if more code is added later on below the if/else.